Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/IndentationStyleConfiguration.swift

```diff
@@ -0,0 +1,61 @@
+import SwiftLintCore
+
+struct IndentationStyleConfiguration: SeverityBasedRuleConfiguration, Equatable {
+	typealias Parent = IndentationStyleRule
+
+	@ConfigurationElement(key: "severity")
+	private(set) var severityConfiguration = SeverityConfiguration<Parent>.warning
+	@ConfigurationElement(key: "include_multiline_strings")
+	private(set) var includeMultilineStrings = false
+	@ConfigurationElement(key: "include_multiline_comments")
+	private(set) var includeMultilineComments = true
+	@ConfigurationElement(key: "per_file")
+	private(set) var perFile = true
```

Comment:
Not sure about this option. I tend to keep it simple for the first and only have a project-wide preferred setting instead of individual variants per file.

Reply:
I've found that many projects I've come across have a hodgepodge mix. I included this so migration to project wide settings can be a little more gradual. Not to mention that Xcode allows setting tabs or spaces on a file by file basis, I don't want to cause violations in areas where teams may have already deliberately gone against the grain.

I can remove, but the work is already done!

Reply:
Got that. Perhaps this can be combined with `preferredStyle` in the sense of having `spaces`, `tabs` and `automatic` (or `perFile`) as options?

This would have the pretty side effect, that the `apply` method can be auto-generated.